### PR TITLE
Add olafurpg to code intel team page

### PR DESCRIPTION
**Pull request by olafurpg**

### CODE REVIEW

**Review by scalabilitysolved — APPROVED** [review]
Looks good :)

**Comment by olafurpg** [comment]
@nicksnyder @sqs I probably rushed too much to merge given change to `README.md`. I'm happy to address any comments with a followup PR

File: company/team/index.md [hunk 1]
```diff
@@ -597,7 +597,7 @@ To add yourself to this page, copy the following template, paste it at the end o
 - [seth@sourcegraph.com](mailto:seth@sourcegraph.com), [LinkedIn](https://www.linkedin.com/in/sethhoover)
 - Seth lives in Menifee, CA with his wife, three dogs, two cats, and a horse (the horse actually has an "apartment" at a nearby equestrian facility). He has been working on the Salesforce.com platform for more than 13 years. He started out working as a member of Salesforce Support prior to moving into consulting. In his free time, he enjoys martial arts, paintball, airsoft, offroading and ATVs, movies and reading.
 
-## Ólafur Páll Geirsson (he/him)
+## Ólafur Páll Geirsson (he/him) <a name="olafurpg"></a>
```
Comment (nicksnyder) [inline company/team/index.md:600]: ah, nice way to create a short link

**Comment by nicksnyder** [comment]
>  I probably rushed too much to merge given change to README.md. I'm happy to address any comments with a followup PR

Totally fine. Bias for action is good!

**Pull request merged**